tools/clean-filestore-limit.sh

```diff
@@ -35,6 +44,7 @@ fi
 
 # See https://cloud.google.com/filestore/docs/troubleshooting#system_limit_for_internal_resources_has_been_reached_error_when_creating_an_instance
 echo "Disabling Filestore API..."
+trap enable_filestore_api EXIT
```

Comment:
Should this go after the next line?  I.e. shouldn't we put the trap after disabling the API?

Reply:
No, you want the trap in place first. It's also harmless.